lib/complete.js

```diff
@@ -228,10 +228,13 @@ class Complete extends EventEmitter {
     if (typeof str !== 'string') return str;
     var shell = (process.env.SHELL || '').split('/').slice(-1)[0];
 
-    var parts = str.split(/(\\)?:/);
-    var name = parts[0];
-    var desc = parts.slice(-1)[0];
-
+    var name = str
+    var desc = ''
+    var matching = /^(.*?)(\\)?:(.*)$/.exec(str)
+    if (matching) {
+      name = matching[1]
+      desc = matching[3]
+    }
```

Comment:
Please add a semi colon, I'm still using them in this project ;)

Don't worry about that, I'll try to merge your PR today and will add them.

Reply:
Finished

Reply:
Awesome, thanks @imcuttle. I'm actually currently working on tabtab, trying to test your PR on bash for now.